pkg/logging/logfields/logfields.go

```diff
@@ -63,6 +63,15 @@ const (
 	// IPAddr is an IPV4 or IPv6 address
 	IPAddr = "ipAddr"
 
+	// IPv4 is an IPv4 address
+	IPv4 = "ipv4"
+
+	// IPv6 is an IPv6 address
+	IPv6 = "ipv6"
```

Comment:
It's not a bit deal but I think we used `ipaddr.ip4` or something like that in other places. We have a ticket to undo that naming scheme, however.

Reply:
I think the above is clean, we should switch to using the above, it's precise and short